flintrock/ec2.py

```diff
@@ -81,6 +81,7 @@ def instances(self):
             return self.slave_instances
 
     @property
+    @functools.lru_cache()
```

Comment:
Nit: Looking at the [docs](https://docs.python.org/3/library/functools.html#functools.lru_cache), I think this can be just `@functools.lru_cache` (without the parentheses).

Reply:
Also done in latest commit ! Many thanks for your review !